dissect/target/plugins/os/windows/_os.py

```diff
@@ -244,13 +244,21 @@ def _part_str(parts: dict[str, Any], name: str) -> str:
         if any(map(lambda value: value is not None, version_parts.values())):
             version = []
 
+            nt_version = _part_str(version_parts, "CurrentVersion")
+            build_version = _part_str(version_parts, "CurrentBuildNumber")
             prodcut_name = _part_str(version_parts, "ProductName")
-            version.append(prodcut_name)
 
-            nt_version = _part_str(version_parts, "CurrentVersion")
+            # CurrentBuildNumber >= 22000 on NT 10.0 indicates Windows 11.
+            # https://learn.microsoft.com/en-us/windows/release-health/windows11-release-information
+            try:
+                if float(nt_version) == 10.0 and int(build_version) >= 22_000:
```

Comment:
```suggestion
                if nt_version == "10.0" and int(build_version) >= 22_000:
```

No need to use the cast to `float()` here, for the `build_version` I can see the use of the `int() >=` construction.